tests/nested/manual/grade-signed-cloud-init-testkeys/task.yaml

```
@@ -140,19 +140,7 @@ execute: |
   # of this test
 
   echo "Waiting for snapd to react to cloud-init"
-  seen=0
-  for i in $(seq 60); do
-      if tests.nested exec "sudo journalctl --no-pager -u snapd" | MATCH "cloud-init reported"; then
-          seen=1
-          break
-      fi
-      sleep 1
-  done
-
-  if [ "$seen" = "0" ]; then
-      echo "snapd did not restrict cloud-init, test failed"
-      exit 1
-  fi
+  tests.nested exec "retry --wait 1 -n 60 sh -c 'sudo journalctl --no-pager -u snapd | MATCH \"cloud-init reported\"'"
```

same comment here, I think we can make it simpler with less nested quoting:

```suggestion
  retry --wait 1 -n 60 sh -c 'tests.nested exec sudo journalctl --no-pager -u snapd | MATCH "cloud-init reported"'
```

---

sure

---

updated